osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/factories/SecurityKeyProvider.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2021 Smart Society Services B.V.
```

Comment:
Copyright header in newly added source files should mention Alliander N.V. instead of Smart Society Service B.V.